models/customer_model.py

```diff
@@ -0,0 +1,30 @@
+from pydantic import BaseModel, EmailStr, Field, root_validator
+from datetime import datetime
+from typing import Optional
+import ulid
+
+
+class Customer(BaseModel):
```

Comment:
acho que seria legal adicionar o campo active que criei pro soft delete, pra quando for da um put em custmer a gente conseguir ativar ele

Reply:
Concordo, ate pra model ter todas as infos da tabela